pt/news/_posts/2016-03-30-ruby-2-1-9-released.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: news_post
+title: "Ruby 2.1.9 lançado"
+author: "usa"
+translator: "fpgentil"
+date: 2016-03-30 12:00:00 +0000
+lang: pt
+---
+
+Ruby 2.1.9 foi lançado.
+
+Esta versão inclui várias correções de *bugs*.
+Veja [ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_1_9/ChangeLog)
```

Comment:
Olá @fpgentil ! Valeu pela contribuição!

Vou ser meio chato, mas bora lá: `Veja ChangeLog para detalhes` soa estranho. Que tal `Consulte o ChangeLog para mais detalhes`? No inglês eles não têm `the` por toda parte, então na hora de traduzir cabe a nós adequar.